string group_name
float32[] target_config
geometry_msgs/Vector3 start
geometry_msgs/Vector3 target
float64 target_tolerance
float64 probe_radius
float64 p
float64 time_limit
uint8 RRTSTAR=0
uint8 TRRT=1
uint8 FMTSTAR=2
uint8 planner_type
---
float64 run_time
float64 total_cost
float64 total_length
geometry_msgs/Vector3 tip_position
geometry_msgs/Vector3 error
uint8 SUCCESS=0
uint8 APPROXIMATE=1
uint8 TIMEOUT=2
uint8 INVALID_START=3
uint8 INVALID_GOAL=4
uint8 INVALID_PARAMETERS=5
uint8 UNKNOWN_ERROR=10
uint8 status
